packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -298,4 +298,9 @@ abstract class CameraPlatform extends PlatformInterface {
   Future<void> dispose(int cameraId) {
     throw UnimplementedError('dispose() is not implemented.');
   }
+
+  /// Sets the output format for the selected camera.
+  Future<void> setFileFormat(int cameraId, ImageFileFormat format) {
```

Comment:
I think we should change this to `setImageFileFormat` to distinguish it from video and image streaming formats.

Reply:
I will add support for other platforms. Only on Android it will still take some time, because I will look at the best way to do this.